### PR TITLE
ROX:25607: disable RHACS default network policies option

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version(s):
4.5+

[Issue](https://issues.redhat.com/browse/ROX-25607)

Link to docs previews:

[secured cluster options, OCP - Helm](https://80234--ocpdocs-pr.netlify.app/openshift-acs/latest/cloud_service/installing_cloud_ocp/install-secured-cluster-cloud-ocp.html#configure-secured-cluster-services-helm-chart-customizations-cloud-ocp) - see "network.enableNetworkPolicies" in table

[secured cluster options, K8s - Helm](https://80234--ocpdocs-pr.netlify.app/openshift-acs/latest/cloud_service/installing_cloud_other/install-secured-cluster-cloud-other.html#secured-cluster-services-config_install-secured-cluster-cloud-other) - see "network.enableNetworkPolicies" in table

[secured cluster options, OCP - Operator](https://80234--ocpdocs-pr.netlify.app/openshift-acs/latest/installing/installing_ocp/install-secured-cluster-config-options-ocp.html#general-and-miscellaneous-settings-secured-cluster_install-secured-cluster-config-options-ocp) - see "network.policies" in table

[secured cluster options, Cloud Service, OCP](https://80234--ocpdocs-pr.netlify.app/openshift-acs/latest/cloud_service/installing_cloud_ocp/install-secured-cluster-cloud-ocp.html#secured-cluster-services-config_install-secured-cluster-cloud-ocp) - see "network.enableNetworkPolicies" in table

[secured cluster options, Cloud Service, K8s](https://80234--ocpdocs-pr.netlify.app/openshift-acs/latest/cloud_service/installing_cloud_other/install-secured-cluster-cloud-other.html#secured-cluster-services-config_install-secured-cluster-cloud-other) - see "network.enableNetworkPolicies" in table

[installing Central on OCP - Operator](https://80234--ocpdocs-pr.netlify.app/openshift-acs/latest/installing/installing_ocp/install-central-ocp.html#install-central-operator_install-central-ocp) - See Step 8, "network"

[Central options, OCP - Operator](https://80234--ocpdocs-pr.netlify.app/openshift-acs/latest/installing/installing_ocp/install-central-config-options-ocp.html#general-and-miscellaneous-settings_install-central-config-options-ocp) - see "network.policies" in table

[installing Central on K8s - Helm](https://80234--ocpdocs-pr.netlify.app/openshift-acs/latest/installing/installing_other/install-central-other.html#default-network-policy-creation_install-central-other)

QE review:
- [x] QE has approved this change. (**ACS has no QE, approved by SME**)

Additional information:

### CODE REVIEW

File: modules/secured-cluster-services-config.adoc [hunk 1]
```diff
@@ -303,6 +303,13 @@ If you do not create this account, you must complete future upgrades manually if
 | `monitoring.openshift.enabled`
 | If you set this option to `false`, {product-title} will not set up {osp} monitoring. Defaults to `true` on {osp} 4.
 
+| `network.enableNetworkPolicies`
+a| To provide security at the network level, {product-title-short} creates default `NetworkPolicy` resources in the namespace where secured cluster resources are installed. These network policies allow ingress to specific components on specific ports. If you do not want {product-title-short} to create these policies, set this parameter to `Disabled`. The default value is `Enabled`.
```
Comment (ebensh) [inline modules/secured-cluster-services-config.adoc:307]: Sorry for the very late review; the Helm chart value is actually a boolean, with the default being true. You can set it to false to disable writing the network policies. (The Operator part with 'Disabled' and 'Enabled' is correct though).

File: modules/central-services-public-config.adoc [hunk 2]
```diff
@@ -76,6 +76,24 @@ When Central, the StackRox Scanner, or Scanner V4 must reach out to services tha
...
+[WARNING]
+====
+Disabling creation of default network policies can break communication between {product-title-short} components. If you disable creation of default policies, you must create your own network policies to allow this communication.
+====
+
+|===
+| Parameter | Description
+
+|`network.enableNetworkPolicies`
+| Specify if {product-title-short} creates default network policies to allow communication between components. To create your own network policies, set this parameter to `Disabled`.
```
Comment (ebensh) [inline modules/central-services-public-config.adoc:93]: Same comment here - for the Helm chart it's a boolean, so either true/false.

**Review by ebensh — CHANGES_REQUESTED** [review]

**Review by jeana-redhat — APPROVED** [review]
LGTM

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Pull request merged**